### PR TITLE
[ci](shellcheck)shellcheck include ssb-tools and tpch-tools

### PR DESCRIPTION
**Pull request by hello-stephen**
# Proposed changes

Issue Number: close #xxx

## Problem summary

according to the comment from https://github.com/apache/doris/pull/11975

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [x] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [x] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by adonis0147** [comment]
Hi @hello-stephen , `ShellCheck` action failed. You can run the script `build-support/shell-check.sh` on your machine to check these issues.

**Comment by hello-stephen** [comment]
> Hi @hello-stephen , `ShellCheck` action failed. You can run the script `build-support/shell-check.sh` on your machine to check these issues.

OK, thanks.
I use VSCode with ShellCheck extension and the check passed yesterday, but today not. :(
maybe I should use build-support/shell-check.sh instead.

**Comment by adonis0147** [comment]
I'm not sure whether the VSCode extension you mentioned supports the configuration `.shellcheckrc`. If it did, I think you could finish the job more easily. Besides, the version of `shellcheck` should be greater than `0.8`.

**Comment by adonis0147** [comment]
Hi @hello-stephen , please rebase the code to re-trigger the P0 test.

File: tools/ssb-tools/bin/build-ssb-dbgen.sh [hunk 1]
```diff
@@ -27,34 +27,34 @@ set -eo pipefail
...
-    echo "Dir $CURDIR/ssb-dbgen/ already exists. No need to download."
+if [[ -d ${SSB_DBGEN_DIR} ]]; then
+    echo "Dir ${CURDIR}/ssb-dbgen/ already exists. No need to download."
     echo "If you want to download ssb-dbgen again, please delete this dir first."
     exit 1
 else
-    cd "$CURDIR"
-    wget https://palo-cloud-repo-bd.bd.bcebos.com/baidu-doris-release/ssb-dbgen-linux.tar.gz && tar -xzvf ssb-dbgen-linux.tar.gz -C $CURDIR/
+    cd "${CURDIR}"
+    wget https://palo-cloud-repo-bd.bd.bcebos.com/baidu-doris-release/ssb-dbgen-linux.tar.gz && tar -xzvf ssb-dbgen-linux.tar.gz -C "${CURDIR}"/
```
Comment (adonis0147) [inline tools/ssb-tools/bin/build-ssb-dbgen.sh:44]: Using `&&` may eat the errors. See #11357 .

File: tools/ssb-tools/bin/build-ssb-dbgen.sh [hunk 2]
```diff
@@ -27,34 +27,34 @@ set -eo pipefail
...
 fi
 
 # compile ssb-dbgen
-cd "$SSB_DBGEN_DIR/" && make
+cd "${SSB_DBGEN_DIR}/" && make
```
Comment (adonis0147) [inline tools/ssb-tools/bin/build-ssb-dbgen.sh:48]: The same as the above.

**Review by adonis0147 — CHANGES_REQUESTED** [review]

**Review by adonis0147 — APPROVED** [review]
LGTM

**Pull request merged**